spec/controllers/content_items_controller_spec.rb

```diff
@@ -2,26 +2,77 @@
 
 RSpec.describe ContentItemsController, type: :controller do
   describe "GET #index" do
-    let(:organisation) { create(:organisation_with_content_items) }
+    let(:organisation) { create(:organisation_with_content_items, content_items_count: 4) }
 
-    before do
-      get :index, params: { organisation_id: organisation }
-    end
+    context "find by organisation" do
+      before do
+        get :index, params: { organisation_id: organisation }
+      end
 
-    it "returns http success" do
-      expect(response).to have_http_status(:success)
-    end
+      it "returns http success" do
+        expect(response).to have_http_status(:success)
+      end
 
-    it "assigns current organisation" do
-      expect(assigns(:organisation)).to eq(organisation)
-    end
+      it "assigns current organisation" do
+        expect(assigns(:organisation)).to eq(organisation)
+      end
+
+      it "assigns list of content items" do
+        expect(assigns(:content_items)).to eq(organisation.content_items)
+      end
 
-    it "assigns list of content items" do
-      expect(assigns(:content_items)).to eq(organisation.content_items)
+      it "renders the :index template" do
+        expect(subject).to render_template(:index)
+      end
     end
 
-    it "renders the :index template" do
-      expect(subject).to render_template(:index)
+    context "sorts with parameters" do
+      context "in ascending order" do
```

Comment:
If you set up the test in the factory (by being implicit) you are basically coupling a factory with a test. This is not a very good practice, in my opinion. You won’t be able, for example, to follow the same approach to test other stuff related to content items and `public_updated_at`.
By being explicit when setting up your test you:
1. Improve readability. You don’t need to go to the factory to understand how the ordering is being set
2. The factory will be simpler. This is a goal by itself.
3. The test will be easier to change.
4. If you remove the test, for example, you don’t need to track the logic that was set in the factory.